.github/workflows/test_installation.yml

```diff
@@ -0,0 +1,26 @@
+name: Test installation process
+
+on:
+  release:
+    types: [created]
```

Comment:
We should probably trigger that on pull requests

Reply:
What's the point of running this job on pull requests if the latest release doesn't change?

Reply:
If it's run on a release it will fail silently unless you explicitly check it, and even if you check it, the release is already in progress

Reply:
It's true, but running it on pull requests doesn't fix this issue because `install` script fetches the latest release anyway. Do you have an idea how to solve this problem?

Also, if the script fails, GitHub should send a notification.

Reply:
We could modify the installation script to accept the gh revision, and run it on pull requests with that parameter

Reply:
In order to make this work, we would have to build, upload and publish protostar. Then trigger this job once the package is ready to download, and modify half of the `install.sh` to support a custom url in order to download that specific package.

Adding such a test doesn't add value because the most fragile part of the script would have to be replaced.